src/Sylius/Bundle/AdminBundle/Controller/DashboardController.php

```diff
@@ -31,7 +30,6 @@ public function __construct(
         private ChannelRepositoryInterface $channelRepository,
         private Environment $templatingEngine,
         private RouterInterface $router,
-        private ?SalesDataProviderInterface $salesDataProvider = null,
```

Comment:
Removed as controllers are excluded from BC and there's no previous deprecation triggering.